products/hashicorp-vault.md

```diff
@@ -16,18 +16,18 @@ versionCommand: vault --version
 releases:
 -   releaseCycle: "1.12"
     eol: false
-    latest: "1.12.3"
-    latestReleaseDate: 2023-02-02
+    latest: "1.12.4"
```

Comment:
Looks like they now use [GitHub releases](https://github.com/hashicorp/vault/releases) instead of [a changelog file](https://github.com/hashicorp/vault/blob/v1.12.4/CHANGELOG.md). The changelogTemplate must be updated.

Reply:
Done: #2614